tests/common.js

```diff
@@ -9,12 +9,26 @@ const Login = async (username, password, page) => {
   await page.locator('#submit_row .loginButton').click();
 };
 
-const getAttributes = async (element) => {
-  const attributes = await element.evaluate(async (ele) => ele.getAttributeNames());
+const getAttributes = async element => {
+  const attributes = await element.evaluate(async ele => ele.getAttributeNames());
   return attributes;
-}
+};
+
+const getNextDay = () => {
+  const tomorrow = new Date();
```

Comment:
Just a suggestion: this variable's name shouldn't be `tomorrow` as Date() will return today's date. Can we have a better name here?